tests/ui-fulldeps/codegen-backend/hotplug.rs

```diff
@@ -6,6 +6,9 @@
 //@ normalize-stdout: "libthe_backend.dylib" -> "libthe_backend.so"
 //@ normalize-stdout: "the_backend.dll" -> "libthe_backend.so"
 
+// Pick a target that requires no target features, so that no warning is shown
+// about missing target features.
+//@ compile-flags: --target arm-unknown-linux-gnueabi
```

Comment:
@bjorn3 does the cranelift backend return anything for `target_features_cfg`? If not, there might be warnings now about missing target features, depending on the ABI info for the current target.

Reply:
Yes, it currently hard codes sse and sse2 for all x86_64 targets that are not bare-metal: https://github.com/rust-lang/rust/blob/7bb98889534d597911263d647424557b414770ec/compiler/rustc_codegen_cranelift/src/lib.rs#L179-L200

Reply:
tests/ui-fulldeps/codegen-backend/ doesn't actually use cg_clif. It uses the backend in tests/ui-fulldeps/codegen-backend/auxiliary/the_backend.rs. It is fine to implement `target_features_cfg` there as always returning sse and sse2. It doesn't compile anything to machine code anyway. It is just a test that `-Zcodegen-backend` with an external codegen backend functions.

Reply:
Yeah I get that. But it still seemed easiest to just use a target that doesn't require any features.